### PR TITLE
issue-660-690

### PR DESCRIPTION
**Pull request by Huygon764**

### CODE REVIEW

**Comment by Huygon764** [comment]
image

**Comment by Huygon764** [comment]
issue 690: change livestyle => lifestyle
isssue 660:  add "Crypto Currencies"

File: packages/app-lixi/src/lang/locales/vi_VN.ts [hunk 1]
```diff
@@ -578,6 +578,7 @@ export default {
   'category.healthAndFitness': 'Sức khỏe và Thể hình',
   'category.travelAndEvents': 'Du lịch và Sự kiện',
   'category.other': 'Khác',
+  'category.cryptoCurrencies': 'Tiền điện tử',
```
Comment (vince8x) [inline packages/app-lixi/src/lang/locales/vi_VN.ts:581]: Please change to "Tiền mã hóa"

Comment (Huygon764) [reply packages/app-lixi/src/lang/locales/vi_VN.ts]: image
Changed

**Review by vince8x — APPROVED** [review]

**Review by vince8x — APPROVED** [review]

**Pull request merged**